nfs-operator.yaml

```diff
@@ -90,7 +90,7 @@ spec:
       containers:
       - name: openebs-provisioner-nfs
         imagePullPolicy: IfNotPresent
-        image: openebs/provisioner-nfs:0.5.0
+        image: openebs/provisioner-nfs:0.6.1
```

Comment:
@mynktl -- can this be replaced with actual version please. `openebs.io/version: dev`

Reply:
Why do we need do that? then there will be inconsistency via helm installation and venial installation, right?

Reply:
done